### PR TITLE
Code cleanups

### PR DESCRIPTION
**Pull request by chanseokoh**

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.dataflow.core.test/src/com/google/cloud/tools/eclipse/dataflow/core/launcher/PipelineLaunchConfigurationTest.java [hunk 1]
```diff
@@ -57,22 +60,14 @@ public void testFromLaunchConfigurationCopiesArgumentsFromLaunchConfiguration()
...
-        ImmutableMap.<String, String>builder()
-            .put("Spam", "foo")
-            .put("Ham", "bar")
-            .put("Eggs", "baz")
-            .build();
-    when(
-        launchConfiguration.getAttribute(
-            PipelineConfigurationAttr.ALL_ARGUMENT_VALUES.toString(),
-            Collections.<String, String>emptyMap())).thenReturn(requiredArgumentValues);
+        ImmutableMap.of("Spam", "foo","Ham", "bar", "Eggs", "baz");
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.dataflow.core.test/src/com/google/cloud/tools/eclipse/dataflow/core/launcher/PipelineLaunchConfigurationTest.java]: space between foo and Ham though I think I find the original version a little easier to read.

File: plugins/com.google.cloud.tools.eclipse.dataflow.core.test/src/com/google/cloud/tools/eclipse/dataflow/core/launcher/PipelineLaunchConfigurationTest.java [hunk 2]
```diff
@@ -86,12 +81,7 @@ public void testToLaunchConfigurationWritesArgumentsToLaunchConfiguration() {
     final PipelineRunner runner = PipelineRunner.DATAFLOW_PIPELINE_RUNNER;
     PipelineLaunchConfiguration lc = PipelineLaunchConfiguration.createDefault();
 
-    Map<String, String> argValues =
-        ImmutableMap.<String, String>builder()
-            .put("Spam", "foo")
-            .put("Ham", "bar")
-            .put("Eggs", "baz")
-            .build();
+    Map<String, String> argValues = ImmutableMap.of("Spam", "foo","Ham", "bar","Eggs", "baz");
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.dataflow.core.test/src/com/google/cloud/tools/eclipse/dataflow/core/launcher/PipelineLaunchConfigurationTest.java]: space between bar and Eggs, if not revert completely

File: plugins/com.google.cloud.tools.eclipse.dataflow.core.test/src/com/google/cloud/tools/eclipse/dataflow/core/launcher/PipelineLaunchConfigurationTest.java [hunk 3]
```diff
@@ -57,22 +60,14 @@ public void testFromLaunchConfigurationCopiesArgumentsFromLaunchConfiguration()
...
     PipelineRunner runner = PipelineRunner.DATAFLOW_PIPELINE_RUNNER;
-    when(
-        launchConfiguration.getAttribute(
-            PipelineConfigurationAttr.RUNNER_ARGUMENT.toString(),
-            PipelineLaunchConfiguration.defaultRunner(majorVersion).getRunnerName()))
-        .thenReturn(runner.getRunnerName());
+    when(launchConfiguration.getAttribute(eq(PipelineConfigurationAttr.RUNNER_ARGUMENT.toString()),
+        anyString())).thenReturn(runner.getRunnerName());
 
     PipelineLaunchConfiguration lc =
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.dataflow.core.test/src/com/google/cloud/tools/eclipse/dataflow/core/launcher/PipelineLaunchConfigurationTest.java]: might want to fix this abbreviated variable name

File: plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/launcher/options/PipelineOptionsNamespaces.java [hunk 4]
```diff
@@ -31,14 +31,13 @@ private PipelineOptionsNamespaces() {}
...
-  private static final Map<MajorVersion, String> VERSION_PREFIXES =
-      ImmutableMap.<MajorVersion, String>builder()
-          .put(MajorVersion.ONE, DATAFLOW_OPTIONS_BASE + ".")
-          .put(MajorVersion.QUALIFIED_TWO, BEAM_OPTIONS_BASE + ".")
-          .put(MajorVersion.TWO, BEAM_OPTIONS_BASE + ".")
-          .put(MajorVersion.THREE_PLUS, BEAM_OPTIONS_BASE + ".")
-          .put(MajorVersion.ALL, BEAM_OPTIONS_BASE + ".")
-          .build();
+  private static final Map<MajorVersion, String> VERSION_PREFIXES = ImmutableMap.of(
+      MajorVersion.ONE, DATAFLOW_OPTIONS_BASE + ".",
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/launcher/options/PipelineOptionsNamespaces.java]: This formatting is easier to follow.

File: plugins/com.google.cloud.tools.eclipse.dataflow.core.test/src/com/google/cloud/tools/eclipse/dataflow/core/launcher/PipelineLaunchConfigurationTest.java (hunk 1 above)
Comment (chanseokoh) [reply plugins/com.google.cloud.tools.eclipse.dataflow.core.test/src/com/google/cloud/tools/eclipse/dataflow/core/launcher/PipelineLaunchConfigurationTest.java]: OK, I can make one pair per line.

**Pull request merged**